tools/test/qa/tests/stability_test/sim.sh

```diff
@@ -0,0 +1,30 @@
+aplay -Dhw:0,1 -f dat -c 2 /dev/zero &
```

Comment:
Each test will need comments at the top describing what it does and any arguments it takes.  Also SPDX style comment stating BSD licence.  The file name here is not very meaningful either, this test does a duplex stream on 0,1 and a playback on 0,0 e.g. "test-duplex-plus-playback.sh" is probably a better name.